### PR TITLE
Issue30 menu styling

### PR DESCRIPTION
**Pull request by trentspi**
Updated the styling of the menu, made the cards a bit bigger, updated a few colors, merged in changes of game icons for the cards...

### CODE REVIEW

File: Page.py [hunk 1]
```diff
@@ -38,10 +38,10 @@ def populate_page (self):
     item_height = self.height / self.col_length
 
     # determine the dimensions of the interior game element
-    game_element_x_pos = int(item_width / 8)
-    game_element_y_pos = int(item_height / 8)
-    game_element_width = int(item_width * 0.75)
-    game_element_height = int(item_height * 0.75)
+    game_element_x_pos = int(25)
+    game_element_y_pos = int(13)
```
Comment (nrhubbar) [inline Page.py]: I am against hard coding these, unless you have a reason

Comment (trentspi) [reply Page.py]: I did this in order to make the game elements border be closer to the edge of the card, but you're right, it should probably be more dynamic based on item_width / item_height. However on a 1920x1080 resolution this works fine (however its likely our end result is not 1920x1080) so I can look into a less "hard coded" solution

Comment (nrhubbar) [reply Page.py]: Thanks, ping me in slack when you got a fix

Comment (rupsis) [reply Page.py]: Do we have a config file that we can pull from?

Comment (trentspi) [reply Page.py]: @rupsis @nrhubbar

Comment (rupsis) [reply Page.py]: I like the idea of loading in style "plug-ins" we can make that a low priority issue.

Comment (jake-white) [reply Page.py]: Should we go ahead and merge this and make a config issue?

**Pull request merged**